maintainers/maintainer-list.nix

```diff
@@ -4666,6 +4666,12 @@
     githubId = 2881922;
     name = "Francis St-Amour";
   };
+  mtrsk = {
+    email = "marcos.schonfinkel@protonmail.com";
+    github = "mtrsk";
+    githubId = 16356569;
+    name = "Marcos Benevides";
+  };
```

Comment:
please split this out into a different commit:
```
maintainers: add mtrsk
```

Reply:
done! :)